docs/core/versions/selection.md

```diff
@@ -27,10 +27,10 @@ The rest of this document examines those four scenarios.
 
 SDK commands include `dotnet new` and `dotnet run`. The `dotnet` CLI must choose an SDK version for every dotnet command. The .NET Core CLI uses the latest SDK installed on the machine by default, even if:
```

Comment:
nit: in the second and third sentence there are "`dotnet` CLI" and ".NET Core CLI". Let's use the same words in both cases. In particular, I suggest to rephrase the second sentence like this (note added backticks at the end of sentence):
> The .NET Core CLI must choose an SDK version for every `dotnet` command.

Reply:
- I agree with @pkulikov. .NET Core CLI is preferable (and the term issued in the top-level [.NET Core command-line interface (CLI) tools](https://docs.microsoft.com/en-us/dotnet/core/tools/?tabs=netcore2x) article. Instead of repeating .NET Core CLI, you could also just use "it" for the second occurrence.
- In "dotnet command", dotnet could be fenced.